db/db_test.go

```diff
@@ -0,0 +1,75 @@
+package db_test
+
+import (
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"testing"
+)
+
+var mockBeer = db.Beer{
+	ID:          1,
+	Desc:        "mock-beer-description",
+	AlcoholCont: 5.5,
+	IBU:         90,
+	SRM:         41,
+	AvgScore:    74.2,
+}
+
+func TestDb(t *testing.T) {
+	//Create
+	b := db.Create(mockBeer)
+
+	if b != mockBeer {
+		t.Errorf("DB creation failed, expecting: %v", mockBeer)
+	}
+
+	if b.ID != int64(1) {
+		t.Errorf("DB creation failed, expecting id to be: 1, recieved: %d", b.ID)
+	}
+
+	//Get
```

Comment:
For good practices, I believe it's better to at least create a new test for each function you are testing.